src/domain/community/application/applicationButton/ApplicationButton.tsx

```diff
@@ -140,7 +141,7 @@ export const ApplicationButton = forwardRef<HTMLButtonElement | HTMLAnchorElemen
       extended
         ? t('components.application-button.extendedMessage', {
             join: verb,
-            journey: journeyLevel > 0 ? t('common.subspace') : t('common.community'),
+            journey: isSpace ? t('common.subspace') : t('common.community'),
```

Comment:
condition was switched here, it should be the other way around (community for space and subspace translation for space)